air_passengers_starting_kit.ipynb

```diff
@@ -539,1527 +648,1398 @@
-       "8900     0     0     0     0     0     0     0     0     0     0     0     0   \n",
-       "8901     0     0     0     0     0     0     0     0     0     0     0     0   \n",
-       "\n",
-       "      d_22  d_23  d_24  d_25  d_26  d_27  d_28  d_29  d_30  d_31  wd_0  wd_1  \\\n",
-       "8897     0     0     0     0     0     0     0     0     0     0     0     0   \n",
-       "8898     0     0     0     1     0     0     0     0     0     0     0     1   \n",
-       "8899     0     0     0     0     0     0     0     0     0     0     0     0   \n",
-       "8900     0     0     0     0     0     0     0     0     0     0     0     0   \n",
-       "8901     0     0     0     0     1     0     0     0     0     0     0     0   \n",
-       "\n",
-       "      wd_2  wd_3  wd_4  wd_5  wd_6  w_1  w_2  w_3  w_4  w_5  w_6  w_7  w_8  \\\n",
-       "8897     0     0     0     0     1    0    0    0    0    0    0    0    0   \n",
-       "8898     0     0     0     0     0    0    0    0    0    0    0    0    0   \n",
-       "8899     0     1     0     0     0    0    0    1    0    0    0    0    0   \n",
-       "8900     0     0     0     0     1    0    0    0    0    1    0    0    0   \n",
-       "8901     0     0     0     1     0    0    0    0    0    0    0    0    0   \n",
-       "\n",
-       "      w_9  w_10  w_11  w_12  w_13  w_14  w_15  w_16  w_17  w_18  w_19  w_20  \\\n",
-       "8897    0     0     0     0     0     0     0     0     0     0     0     0   \n",
-       "8898    0     0     0     0     0     0     0     0     0     0     0     0   \n",
-       "8899    0     0     0     0     0     0     0     0     0     0     0     0   \n",
-       "8900    0     0     0     0     0     0     0     0     0     0     0     0   \n",
-       "8901    0     0     0     0     0     0     0     0     0     0     0     0   \n",
-       "\n",
-       "      w_21  w_22  w_23  w_24  w_25  w_26  w_27  w_28  w_29  w_30  w_31  w_32  \\\n",
-       "8897     0     0     0     0     0     0     0     0     0     0     0     0   \n",
-       "8898     0     0     0     0     0     0     0     0     0     0     0     0   \n",
-       "8899     0     0     0     0     0     0     0     0     0     0     0     0   \n",
-       "8900     0     0     0     0     0     0     0     0     0     0     0     0   \n",
-       "8901     0     0     0     0     0     0     0     0     0     0     0     0   \n",
-       "\n",
-       "      w_33  w_34  w_35  w_36  w_37  w_38  w_39  w_40  w_41  w_42  w_43  w_44  \\\n",
-       "8897     0     0     0     0     0     0     1     0     0     0     0     0   \n",
-       "8898     0     0     0     0     0     0     1     0     0     0     0     0   \n",
-       "8899     0     0     0     0     0     0     0     0     0     0     0     0   \n",
-       "8900     0     0     0     0     0     0     0     0
```

Comment:
I would ahve used

%%file 

here to be sure people edit the files while using the notebook.

Reply:
and I still see references to ramp_test_submission

Reply:
The `%%file` can be messy for our CIs. Also the purpose was to show how to write the submission `estimator.py` file - not for them to edit it in the notebook. But if you prefer I can amend this and the explanation.

`ramp_test_submission` is still there because we ask them to test locally before submitting.

@agramfort

Reply:
Ignore `ramp_test_submission` @glemaitre explained to me.